src/interfaces.ts

```diff
@@ -47,6 +66,11 @@ export interface Row extends Array<any> {
   [index: number]: string | number;
 }
 
+export interface KeyVal {
```

Comment:
I think maybe a tuple type here might be nicer? And while we're at it, we might as well make it generic rather than using any?

```typescript
export type KeyVal<T = any> = [string, T];
```

This allows us to do things like the following, which will also catch things like trying to index out of range etc.

```typescript
const one: KeyVal<string> = ["a", "b"];
console.log(one[0]);
```

Reply:
We _should_ also make `RpcReceipt` generic, because we'll know exactly the response type we're expecting given the method we're calling:

```typescript
export interface RpcReceipt<T = any> {
  jsonrpc: string;
  id: number;
  result: T;
}
```

Note also that this simplifies some of the types:

```typescript
export type Column = Array<ColumnDescriptor>;

export type Row = Array<string | number>;
```